client/js/main.js

```diff
@@ -5,14 +5,29 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
-const app = new Vue({
-  vuetify,
-  store,
-  router,
-  render: (h) => h(App),
-}).$mount('#app');
+async function initApp() {
+  // Make sure we try and reestablish a player's session if one exists
+  // We do this before the app mounts to preempt the reauth/logout logic
+  try {
+    await store.dispatch('requestStatus');
+  } catch {
+    // Unable to validate player status or player not authenticated
+    // Swallow the error to let the player continue
+  }
 
-// Expose app for testing
-if (window.Cypress) {
-  window.app = app;
+  const app = new Vue({
+    vuetify,
+    store,
+    router,
+    render: (h) => h(App),
+  }).$mount('#app');
+
+  // Expose app for testing
+  if (process.env.NODE_ENV === 'development' || window.Cypress) {
```

Comment:
Here's a question: am I right that the node env vars are read-only to the application? Maybe we should only check that the mode is development so that it's harder to force the application to expose its internals, since `process.env.NODE_ENV` is set during build time and theoretically can't be overwritten by the client. What do you think? This would however limit the e2e test suite to `development` mode, which to be fair is generally what we do anyway. Just food for thought

```suggestion
  if (process.env.NODE_ENV === 'development') {
```